### ISSUE TITLE
Error "sign up not completed"

### ISSUE THREAD

**Issue by AlanSGC** [tomgilder/flutter_sign_in_with_apple#37]
Hello,
I've been using this plugin in my project for a few months now, everything working fine, including published in the Apple Store. However, this week, when trying to upload a new version to the Apple Store, the app was rejected in the Apple review process, with the error "Sign up not completed", and we did not make any changes to the sign in apple process.
Has anyone had this problem?
Thanks

**Comment by verilune** [comment]
I submitted my app last week, it got rejected because they asked me to add Apple Sign In. Now I have added it and tested it on Debug and Release version. 2 other testers have tested the release version from Testflight. However the application got rejected, the reviewer says that he cannot login on iPad or Iphone with iOS 13.4. I updated my iPhone to iOS 13.4 and it works on my device.

* We used Touch ID in testing
* People are reporting Apple Sign In issues with Face ID
* Not sure what Apple Reviewers use for ID type.

**Comment by luccascorrea** [comment]
Hi, I'm having the same problem. Did any of you figure out how to solve this @AlanSGC and @verilune? Thanks

**Comment by verilune** [comment]
It could be that reviewers use different version of iOS or the have VPN or something that fails the test?! Is it working for you but not for reviewer?

**Comment by luccascorrea** [comment]
Yes, I was thinking it could be something like that. And you're right, it's working fine for me but not for them. Quite frustrating

**Comment by verilune** [comment]
I had the same issue, ask a couple of testers to test Apple Sign In on Testflight version/release, if it works for them too, record your successful test, upload to youtube(private video). Request a new review with youtube link as proof + test result from other testers in review notes.

**Comment by luccascorrea** [comment]
That's a good idea. I will do that. Thank you
